src/components/BarChart/BarChart.tsx

```diff
@@ -235,6 +238,17 @@ export const BarChart = ({
     [activeBar, loading, rangeMax, rangeMin, value]
   )
 
+  // disables the behavior of chanhing value while scrolling on top of the input
+  const onRangeWheel = useCallback((e) => {
+    // Prevent the input value change
+    e.target.blur()
+    // Prevent the page/container scrolling
+    e.stopPropagation()
+    setTimeout(() => {
```

Comment:
WDYT about cleaning the timeout when un-mounting the component?

Reply:
for sure, added!